spec/factories.rb

```diff
@@ -63,12 +63,12 @@
     suspension_period_end_date { nil }
     blocking_period_start_date { nil }
     blocking_period_end_date { nil }
+    order_number { attributes_for(:order_number) }
   end
 
   factory :measure do
     transient do
       measure_type_description { Forgery(:basic).text }
-      ordernumber { Forgery(:basic).number(exactly: 5) }
```

Comment:
Why removing this? 
Will measure mocks never be used on their own and always be initialised as children of something else that will pass down this value?

Reply:
So this is just an unused transient attribute. Probably someone was tempted to recreate presenting a quota measure with an order number but failed to find a use for this.